docs/howto/operate/new-cluster/new-cluster.md

```diff
@@ -130,7 +139,7 @@ For example, to work with Pangeo you would initialise terraform like so:
 terraform init -backend-config=pangeo-backend.hcl -reconfigure
 ```
 
-TODO: add instructions on how/when to create other backends
+<!-- TODO: add instructions on how/when to create other backends -->
```

Comment:
Just a note that the `%` denotes a comment in MyST, so you could do `% TODO: etc etc` as well.

Reply:
Can we add various markdown/rST/MySt syntaxes to the training issue as well please? :sweat_smile: